styles.css

```diff
@@ -1,31 +1,62 @@
-* {
+body {
     margin: 0;
     padding: 0;
-    box-sizing: border-box;
+    background: #f3f3f3;
 }
 
-body {
-    background-color: #bbb;
+.wrapper {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: 100%;
+    transform: translate(-50%, -50%);
+    text-align: center;
 }
 
-.powitanie {
-    height: 40vh;
-    margin: 100px;
-    text-align: center;
-    line-height: 10vh;
-    font-family: 'Bitter', serif;
-    background-color: #fff;
-    box-shadow: 3px 3px 10px, -3px 3px 10px, 3px -3px 10px;
+.custom-checkbox {
+    display: inline-block;
+    position: relative;
+    width: 40px;
+    height: 20px;
+    background: #c6c6c6;
+    border-radius: 20px;
+    box-shadow: inset 0 0 5px rgba(0, 0, 0, .2);
+}
+
+.custom-checkbox::before {
+    content: '';
+    display: inline-block;
+    width: 20px;
+    height: 20px;
+    background: #fff;
+    border-radius: 10px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    transform: scale(1.1);
+    box-shadow: 0 2px 5px rgba(0, 0, 0, .2);
+    transition: .5s left ease-in-out;
 }
 
-h1 {
-    padding-top: 70px;
-    letter-spacing: 2px;
-    word-spacing: 4px;
+.is-visually-hidden {
+    position: absolute !important;
+    height: 1px;
+    width: 1px;
+    overflow: hidden;
+    clip: rect(1px 1px 1px 1px);
+    IE6,
```

Comment:
Remove those IE6/IE7 from here, this is not valid CSS. This should be placed as comment instead: `/** IE6 and IE7 support **/ 
Or remove this and line 48 completely